docs/kms/README.md

```diff
@@ -14,6 +14,43 @@ Vault as Key Management System requires following to be configured in Vault
 - AppRole based authentication with read/update policy for transit backend. In particular, read and update policy
   are required for the generate data key endpoint and decrypt key endpoint.
 
+Here is a sample quick start to configuring vault with a transit backend and Approle with correct policy 
+#### 2.1 Start vault server in dev mode 
+Note that this is insecure and stores all its data in memory,and not meant for production
+```
+vault server -dev
+```
+
+#### 2.2 Set up vault transit backend and  create an app role
+```
+cat > vaultpolicy.hcl <<EOF
+path "transit/datakey/plaintext/my-minio-key" { 
+  capabilities = [ "read", "update"]
+}
+path "transit/decrypt/my-minio-key" { 
+  capabilities = [ "read", "update"]
+}
+path "transit/encrypt/my-minio-key" { 
+  capabilities = [ "read", "update"]
+}
+
+EOF
+
+export VAULT_ADDR='http://127.0.0.1:8200'
+vault auth enable approle    # enable approle style auth
+vault secrets enable transit  # enable transit secrets engine
+vault write -f  transit/keys/my-minio-key  #define a encryption key-ring for the transit path
+vault policy write minio-policy ~/code/src/github.com/minio/vaultpolicy.hcl  #define a policy for AppRole to access transit path
+vault write auth/approle/role/my-role token_num_uses=0  secret_id_num_uses=0  period=60s # period indicates it is renewable if tok renewed before the period is over
+# define an AppRole
+vault write auth/approle/role/my-role policies=minio-policy # apply policy to role
+vault read auth/approle/role/my-role/role-id  # get Approle ID
+vault write -f auth/approle/role/my-role/secret-id
+
+```
+
+The AppRole ID, AppRole Secret Id, Vault endpoint and Vault key name can now be used to start minio server with Vault as KMS.
+
 ### 3. Environment variables
 
 You'll need the Vault endpoint, AppRole ID, AppRole SecretID, encryption key-ring name before starting Minio server with Vault as KMS
```

Comment:
Should provide sample snippet with output that users can verify things are successfully working

Reply:
"...AppRole ID, AppRole SecretID and encryption key-ring name..."